tests/parsers/test_state.py

```diff
@@ -96,3 +92,10 @@ def test_offense_record_row__enabled(section, report, state):
     state.offense_num = 1
     state.section = section
     assert offense.OffenseRecordRow(report, state).is_enabled()
+
+
+def test_find__not_enabled(report, state):
```

Comment:
Extra underscore?

Reply:
I was trying to follow the same naming scheme of the other tests in this file. Maybe I misunderstood why the double underscore was there for other tests. Let me know if it should be changed